### DISCUSSION TITLE
dyo up its running only localhost

### DISCUSSION THREAD

**Discussion by FrancisVarga** [dyrector-io/dyrectorio#1005] [Q&A]
Hi how can i change the config so it can list to normal ip

**Comment by polaroi8d** [comment]
@FrancisVarga are you using the `CLI` or `docker-compose`?

**Comment by polaroi8d** [comment]
Based on that, we’ve reworked the CLI and are now primarily focusing on using docker-compose to set up the dyrectorio platform locally. I’m closing this issue.
